android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -236,6 +238,8 @@ private void initializeUpdateAfterRestart() {
                     // Therefore, deduce that it is a broken update and rollback.
                     rollbackPackage();
                 } else {
+                    // Clear the React dev bundle cache so that new updates can be loaded.
+                    clearReactDevBundleCache();
```

Comment:
It might not be entirely necessary, but I'd still love to know whether we can do this only during debug builds. I'd like to be hyper-sensitive with any work we do as part of the app load process, to keep CodePush as impactless as possible.

Could you quickly try seeing whether `BuildConfig.DEBUG` can be used here? I'm assuming that it would be true/false depending on what the "parent" project configuration is, and therefore, when the dev support is enabled, we would clear the bundle, but when it's not, we wouldn't.